scripts/tests/deployAndEnableConnector.ts

```diff
@@ -0,0 +1,45 @@
+import { addresses as addressesPolygon } from "./polygon/addresses";
+import { addresses } from "./mainnet/addresses";
+import { abis } from "../constant/abis";
+
+import hre from "hardhat";
+import type { Signer, Contract } from "ethers";
+import type { ContractJSON } from "ethereum-waffle/dist/esm/ContractJSON";
+
+const { ethers, waffle } = hre;
+const { deployContract } = waffle;
+
+interface DeployInterface {
+  connectorName: string;
+  contractArtifact: ContractJSON;
+  signer: Signer;
+  connectors: Contract;
+}
+
+function getAddress(network: string | undefined) {
+  if (network === "polygon") return addressesPolygon;
+  // else if (network === "arbitrum") return addressesPolygon;
+  // else if (network === "avalanche") return addressesPolygon;
```

Comment:
@pradyuman-verma Let's uncomment these and add avalanche and arbitrum constants?

Reply:
@pradyuman-verma Still commented?

Reply:
ohh, forgot this one

Reply:
done and pushed.